### PR TITLE
Solutions of problems from lesson 1

### PR DESCRIPTION
**Pull request by BartoszPi**
Solutions for problems on Snakify: 1-6 for lesson 1 and 1-9 for lesson 2.

@gradopaw please review

### CODE REVIEW

File: students/pietrewicz_bartosz/lesson_01_basics/1_6.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Read the number of students in 3 classes
+a = int(input())
+b = int(input())
+c = int(input())
+# Print the minimal number of desks the school must purchase
+desks_a = a//2 + a%2
```
Comment (gradopaw) [inline students/pietrewicz_bartosz/lesson_01_basics/1_6.py]: PEP8 - missing whitespace around modulo operator

File: students/pietrewicz_bartosz/lesson_01_basics/1_6.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+desks_b = b//2 + b%2
```
Comment (gradopaw) [inline students/pietrewicz_bartosz/lesson_01_basics/1_6.py]: PEP8 - missing whitespace around modulo operator

File: students/pietrewicz_bartosz/lesson_01_basics/1_6.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+desks_c = c//2 + c%2
```
Comment (gradopaw) [inline students/pietrewicz_bartosz/lesson_01_basics/1_6.py]: PEP8 - missing whitespace around modulo operator

File: students/pietrewicz_bartosz/lesson_01_basics/2_1.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# Read an integer
+a = int(input())
+# Print the last digit of the integer
+print(a%10)
```
Comment (gradopaw) [inline students/pietrewicz_bartosz/lesson_01_basics/2_1.py]: PEP8 - missing whitespace around modulo operator

File: students/pietrewicz_bartosz/lesson_01_basics/2_3.py [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# Read a 3-digit integer
+a = int(input())
+# Print the sum of its digits
+print(a%10 + a//10%10 + a//100)
```
Comment (gradopaw) [inline students/pietrewicz_bartosz/lesson_01_basics/2_3.py]: PEP8 - missing whitespace around modulo operator

File: students/pietrewicz_bartosz/lesson_01_basics/2_2.py [hunk 6]
```diff
@@ -0,0 +1,4 @@
+# Read an integer
+a = int(input())
+# Print its tens digit
+print(a//10%10)
```
Comment (gradopaw) [inline students/pietrewicz_bartosz/lesson_01_basics/2_2.py]: PEP8 - missing whitespace around modulo operator

File: students/pietrewicz_bartosz/lesson_01_basics/2_5.py [hunk 7]
```diff
@@ -0,0 +1,4 @@
+# Read a real number
+a = float(input())
+# Print first digit of the number to the right of decimal point
+print(int(a*10)%10)
```
Comment (gradopaw) [inline students/pietrewicz_bartosz/lesson_01_basics/2_5.py]: PEP8 - missing whitespace around modulo operator

File: students/pietrewicz_bartosz/lesson_01_basics/2_7.py [hunk 8]
```diff
@@ -0,0 +1,4 @@
+# Read the number of minutes passed since midnight
+a = int(input())
+# Print the hour and minutes
+print(a//60, a%60)
```
Comment (gradopaw) [inline students/pietrewicz_bartosz/lesson_01_basics/2_7.py]: PEP8 - missing whitespace around modulo operator

File: students/pietrewicz_bartosz/lesson_01_basics/2_8.py [hunk 9]
```diff
@@ -0,0 +1,7 @@
+# Read the number of dollars (A) and cents (B) the cupcake costs
+# and the number of cupcakes (N)
+A = int(input())
+B = int(input())
+N = int(input())
+# Print the summarized cost of N cupcakes
+print(N*A+N*B//100, N*B%100)
```
Comment (gradopaw) [inline students/pietrewicz_bartosz/lesson_01_basics/2_8.py]: PEP8 - missing whitespace around modulo operator

**Comment by BartoszPi** [comment]
@gradopaw please review

**Comment by BartoszPi** [comment]
@gradopaw please review

**Comment by BartoszPi** [comment]
@gradopaw please review

**Comment by BartoszPi** [comment]
@gradopaw please review

**Pull request merged**